backend/app/analysis/sentiment_analysis.py

```diff
@@ -0,0 +1,35 @@
+import nltk
+from nltk.sentiment import SentimentIntensityAnalyzer
+import numpy as np
+
+
+def text_to_sound(user_text):
+    """
+    Takes text, runs NLTK's sentiment analyzer, and then returns a sound object
+    """
+
+    nltk_analyzer = SentimentIntensityAnalyzer()
+    all_sentences = nltk.sent_tokenize(user_text)
+
+    all_scores = []
+    for each_sentence in all_sentences:
+        all_scores.append(nltk_analyzer.polarity_scores(each_sentence)["compound"] * 15)
```

Comment:
What's the significance of this `15`? Could we pull this out and assign it as a constant with a descriptive name?